code/src/pages/products/ProductsPage.test.tsx

```diff
@@ -41,7 +41,7 @@ jest.mock("@/store/api/productsApi", () => ({
 }));
 
 const defaultQueryArguments = {
-  size: 10,
+  size: Infinity,
```

Comment:
Why we need `Infinity` here?

Reply:
This is just a temporary solution. It is made so, i can pull list of all products from db, because currently db requires size of the elements that we get from DB. When we implement endless scrolling, then we will go back to using a realistic value for that DB request.

Reply:
Okay, I got it. So please add comment on the right of this line to avoid confusion among other developers when they work with this code

Reply:
Done